test-utils/src/androidLibMain/kotlin/mock/permissions/MockPermissionContext.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ Copyright 2021 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.test.mock.permissions
+
+import android.content.Context
+import com.splendo.kaluga.permissions.PermissionContext
+import org.mockito.Mockito
+
+actual val mockPermissionContext = PermissionContext(Mockito.mock(Context::class.java))
```

Comment:
Could also use `MockContext`, not sure if the mockito dependency is needed by anything else at the moment.

Reply:
MockContext is only available for instrumented tests afaik. At least, I wasnt able to import it